mne/viz/_3d.py

```diff
@@ -1787,6 +1804,16 @@ def plot_volume_source_estimates(stc, src, subject=None, subjects_dir=None,
     %(transparent)s
     show : bool
         Show figures if True. Defaults to True.
+    initial_time : float | None
+        The initial time to plot. Can be None (default) to use the time point
+        with the maximal absolute value activation.
```

Comment:
What happens if `initial_time` is `None` and `initial_pos` is not and vice-versa?

Reply:
Pushed a commit to explain